miden-tx/src/host/testing/utils.rs

```diff
@@ -1,8 +1,3 @@
-#![no_std]
-
-#[macro_use]
-extern crate alloc;
-
 #[cfg(feature = "std")]
 extern crate std;
```

Comment:
don't we still need this?

Reply:
What do you mean? The commented code was not modified

Reply:
Oh the deleted macros are meant to be placed on the root of the crate

Reply:
ah yeah, I meant this because I saw the other macros. But I was wondering if the `extern crate std` was still needed on that particular file/on the root of the crate or not.

Reply:
For `miden-tx`, this is already there on the root.

Reply:
in that case don't we need to remove it from here (`miden-tx/src/host/testing/utils.rs`) then?

Reply:
Oh I had misunderstood your previous comment, yes this can be removed here! Done